src/usecase_regression.Rmd

```diff
@@ -0,0 +1,107 @@
+# Regression
+For the regression use case we use the well-known [Boston Housing](https://www.rdocumentation.org/packages/mlbench/versions/0.4-1/topics/BostonHousing) dataset. The road map is as follows:
+
+* define the learning task  ([here](http://mlr-org.github.io/mlr-tutorial/devel/html/task/index.html)),
+* tune the model ([here](http://mlr-org.github.io/mlr-tutorial/devel/html/tune/index.html)),
+* conduct a benchmark experiment ([here](http://mlr-org.github.io/mlr-tutorial/devel/html/benchmark_experiments/index.html)) and
+* evaluate the performance of the model ([here](http://mlr-org.github.io/mlr-tutorial/devel/html/performance/index.html)).
+
+First, we have a look at a part of the data.
+```{r}
+data(BostonHousing, package = "mlbench")
+head(BostonHousing)
+```
+We have 14 numeric variables and as before ``medv`` is chosen to be the response variable for the regression model. 
```

Comment:
Consider starting a new sentence here, imo more readable:
The dataset contains 14 numeric variables. As before, ...